vapi/tags/tags.go

```diff
@@ -106,13 +109,19 @@ func (c *Manager) UpdateTag(ctx context.Context, tag *Tag) error {
 		},
 	}
 	url := c.Resource(internal.TagPath).WithID(tag.ID)
-	return c.Do(ctx, url.Request(http.MethodPatch, spec), nil)
```

Comment:
just an error

Reply:
Resolved in https://github.com/vmware/govmomi/commit/d14cbf39ee092c1a12388ab5b8009bc3c443c898.